feature-group-definitions/media-session.yml

```
@@ -0,0 +1,25 @@
+name: Media session
+description: The `navigator.mediaSession` API integrates with platform UI for media playback. It can be used to set metadata such as title and artwork, and to handle user actions like playing, pausing, or seeking.
```

I'm not sure what "integrates with platform UI" means and the spec is not helping me here. Do you know more about this? What does platform UI refer to?

---

It's mainly mobile phone lock screen controls for audio playback which show the title and album art, and controls from play/pause/prev/next. Desktop operating systems can also have a thing like this, but I've not interacted with those myself.

---

How about this then?

```suggestion
description: The `navigator.mediaSession` API integrates with device UI for media playback. It can be used to set metadata, such as title and artwork, and to handle user actions, like playing, pausing, or seeking.
```

---

Implying, I guess that "device" would be our preferred term for "device or operating system"

---

That WFM.